src/custom-types.json

```diff
@@ -347,7 +347,11 @@
   "application/x-pkcs12": {
     "compressible": false
   },
-  "application/x-rar-compressed": {
+  "application/x-protobuf": {
+    "compressible": true,
+    "sources": ["https://developers.google.com/api-client-library/java/google-http-java-client/reference/1.20.0/com/google/api/client/protobuf/ProtocolBuffers"]
+  },
+    "application/x-rar-compressed": {
```

Comment:
Indent error here

Reply:
not sure what you mean here - I changed the url to be in a new line if that's what you mean

Reply:
It seems like there are four spaces now on this line when there used to be only 2 spaces.